src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -1796,51 +1800,57 @@ private void CheckForSolutionFile()
 				.FirstOrDefault()?.ItemPath;
 		}
 
+		private void GetDesktopIniFileData()
+		{
+			var path = Path.Combine(WorkingDirectory, "desktop.ini");
+			DesktopIni = WindowsIniService.GetData(path);
+		}
+
 		public void CheckForBackgroundImage()
 		{
-			var iniPath = Path.Combine(WorkingDirectory, "desktop.ini");
-			if (!File.Exists(iniPath))
+			var filesAppSection = DesktopIni.FirstOrDefault(x => x.SectionName == "FilesApp");
+			if (filesAppSection is null)
 				return;
```

Comment:
Before returning, `FolderBackgroundImageSource` should be set to null to clear out any existing image.

Reply:
Done.